codes/csharp/csharp.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
Great to add the VS sln file!

Is this compatible with previous VS or only VS 2022?